afterpay/src/main/kotlin/com/afterpay/android/view/AfterpayPriceBreakdown.kt

```diff
@@ -66,8 +65,8 @@ class AfterpayPriceBreakdown @JvmOverloads constructor(
     // The terms and conditions are tied to the configured locale on the configuration
     private val infoUrl: String
         get() {
-            val country = Afterpay.locale.country.toLowerCase(Locale.ROOT)
-            return "https://static-us.afterpay.com/javascript/modal/${country}_rebrand_modal.html"
+            val locale = "${Afterpay.locale.language}_${Afterpay.locale.country}"
```

Comment:
Should there be a data type specified for this variable?

```suggestion
            val locale: String = "${Afterpay.locale.language}_${Afterpay.locale.country}"
```

Reply:
> Should there be a data type specified for this variable?

It's recommended to declare type explicitly if a property forms part of the public interface or if the declaration cannot provide enough information for the type to be inferred, otherwise the above (especially for local variables) is idiomatic.